testing/examples/test_blocks.rs

```diff
@@ -49,13 +49,13 @@ fn mine_block<S: Storage>(
     let old_block_height = miner.consensus.ledger.get_current_block_height();
 
     // add it to the chain
-    miner.consensus.receive_block(&block)?;
+    miner.consensus.receive_block(&block).await?;
 
     let new_block_height = miner.consensus.ledger.get_current_block_height();
     assert_eq!(old_block_height + 1, new_block_height);
 
     // Duplicate blocks dont do anything
-    miner.consensus.receive_block(&block)?;
+    miner.consensus.receive_block(&block).await.ok(); // throws a duplicate error -- seemingly intentional
```

Comment:
it's a very old test, so perhaps it's an old API where duplicate blocks hadn't returned an error yet